package.json

```diff
@@ -28,7 +28,9 @@
     "packages/*"
   ],
   "devDependencies": {
+    "@playwright/test": "^1.39.0",
     "@tapjs/nock": "^3.1.13",
+    "@types/node": "^20.8.7",
```

Comment:
since we target Node 18 as our base version, we should keep the @types in line with Node 18